server/ipqueue.go

```diff
@@ -130,14 +130,16 @@ func (q *ipQueue[T]) push(e T) (int, error) {
 		q.elts = *(q.pool.Get().(*[]T))
 	}
 	q.elts = append(q.elts, e)
+	1++
```

Comment:
This is a no-op as it's a numeric `1`, not a letter `l`.

Reply:
And that would be wrong anyway since we check after that for `if l == 0` to know if we need to signal.